src/services/utils/IssueUtil.ts

```diff
@@ -74,3 +96,88 @@ export function checkIsInAssignFlow(
 
   return hasMentor;
 }
+
+export function updateNotification(
+  message: string,
+  issueBody: string,
+  sender?: string
+): string {
+  const reg = /## :warning:Notification:warning:/i;
+  if (!reg.test(issueBody)) {
+    issueBody = issueBody + IssueNotificationHead.NOTIFICATION;
+  }
+  let newMessage: string = message;
+  if (sender !== undefined) {
+    newMessage = "@" + sender + " " + message;
+  }
+
+  const notificationData = issueBody.match(NOTIFICATION_REGEX); //new notification
+  if (notificationData?.length !== 3) {
+    return issueBody;
+  }
+
+  const newIssuebody = issueBody.replace(notificationData[2], newMessage); //update message
+  return newIssuebody;
+}
+
+export function updateStatus(
+  issueBody: string,
+  challenger?: string,
+  program?: string
+): string {
+  const reg = /## Status/i;
+  if (!reg.test(issueBody)) {
+    issueBody = issueBody + IssueNotificationHead.STATUS;
+  }
+
+  let newMessage: string;
+
+  if (program === undefined && challenger === undefined) {
+    newMessage = `&nbsp;&nbsp; The challenge has not picked yet.
+`;
+  } else if (program == undefined) {
+    newMessage = `&nbsp;&nbsp;Current challenger: @${challenger}
+`;
+  } else {
+    newMessage = `&nbsp;&nbsp;Current challenger: @${challenger}
+&nbsp;&nbsp;Current Program: ${program}
+`;
+  }
+
+  const StatusData = issueBody.match(STATUS_REGEX); //new notification
+  if (StatusData?.length !== 3) {
+    return issueBody;
+  }
+
+  const newIssuebody = issueBody.replace(StatusData[2], newMessage); //update message
+  return newIssuebody;
+}
+
+export async function createOrUpdateNotification(
```

Comment:
I think this function should not be here. Because IssueUtil only contains some general and stateless functions.

Reply:
I think it should be in the command API. Because the context comes form there.

Reply:
I think it should be, I'll move them to the right place.(maybe `src/commands/issue-update`)

Reply:
I think the update the issue not a command(like pick-up/gitve-up).